tests/chainer_tests/training_tests/updaters_tests/snippets/cuda_init.py

```diff
@@ -0,0 +1,40 @@
+import numpy
+
+import chainer
+from chainer.backends import cuda
+from chainer import testing
+import chainer.training.updaters.multiprocess_parallel_updater as mpu
+
+
+def test():
+    model = chainer.Link()
+    dataset = [((numpy.ones((2, 5, 5)) * i).astype(numpy.float32),
+                numpy.int32(0)) for i in range(100)]
+
+    batch_size = 5
+    devices = (0,)
+    iters = [chainer.iterators.SerialIterator(i, batch_size) for i in
+             chainer.datasets.split_dataset_n_random(
+                 dataset, len(devices))]
+    optimizer = chainer.optimizers.SGD(lr=1.0)
+    optimizer.setup(model)
+
+    # Initialize CUDA context.
+    cuda.cupy.cuda.runtime.runtimeGetVersion()
+
+    try:
+        updater = mpu.MultiprocessParallelUpdater(
```

Comment:
./tests/chainer_tests/training_tests/updaters_tests/snippets/cuda_init.py:26:9: F841 local variable 'updater' is assigned to but never used